src/nusystematics/app/DumpConfiguredTweaksNuSyst.cxx

```diff
@@ -577,13 +600,27 @@ int main(int argc, char const *argv[]) {
                 << std::flush;
     }
 
-    tst.Clear();
+    
 
     // Calcuate weights
     if(RunNuSyst){
       event_unit_response_w_cv_t resp = phh.GetEventVariationAndCVResponse(GenieGHep);
       tst.Add(resp);
     }
+    // --- Compute W from (q0, Q2) and nucleon mass *right before Fill* ---
+    //    W^2 = M_N^2 + 2 M_N q0 - Q^2  (q0 = energy transfer)
+    {
```

Comment:
I don't think we need `{...}` block here. Also, move this above `if(RunNuSyst){`